topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md

```diff
@@ -0,0 +1,482 @@
+
+> <details-title> Cosine score methods </details-title>
+>
+> One of the most common methods to compute the similarity score between two spectra is computing the cosine of the *angle* between them. Each spectrum can be represented as a vector with an axis along each of its *m/z* values. Then the cosine of the angle is basically a normalised *dot product* of these two vectors. The intensity and *m/z* weighting corrections have been used to optimise performance. For example, higher mass peaks can be given greater weight since they are more discriminating than lower mass peaks. *Reverse* scores assign no weight to peaks present only in the measured sample spectrum under the assumption that they arise from impurities.
+>
+{: .details}
+
+> <hands-on-title> Compute similarity scores </hands-on-title>
+>
+> 1. {% tool [matchms similarity](toolshed.g2.bx.psu.edu/repos/recetox/matchms/matchms/0.17.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Queries spectra"*: `RI using kovats of Mass spectra from RAMClustR` (output of **RIAssigner** {% icon tool %})
+>    - *"Symmetric"*: `No` (if we were to query our spectra agains itself, we would select `Yes`)
+>        - {% icon param-file %} *"Reference spectra"*: `reference_spectral_library.msp` (downloaded file from Zenodo)
+>    - In *"Algorithm Parameters"*:
+>        - *"tolerance"*: `0.03`
+>    - *"Apply RI filtering"*: `Yes` (we use this since our data is GC)
+>
+>    You can leave the other parameters with their default values.
+>
+>    > <comment-title> Used reference spectra </comment-title>
+>    > The RECETOX Metabolome HR-[EI+]-MS library is a collection of mostly endogenous compounds from the MetaSci Human Metabolite Library. Analytes underwent methoximation/silylation prior to acquisition. Spectra were acquired at 70 eV on Thermo Fisher Q Exactive™ GC Orbitrap™ GC-MS/MS at 60000 resolving power.
+>    {: .comment}
+>
+{: .hands_on}
+
+> <details-title> Overview of the spectral similarity scores </details-title>
+> > <h5>Cosine Greedy</h5>
+> > The cosine score, also known as the dot product, is based on representing the similarity of two spectra through the cosine of an angle between the vectors that the spectra produce. Two peaks are considered as matching if their *m/z* values lie within the given tolerance. Cosine greedy looks up matching peaks in a "greedy" way, which does not always lead to the most optimal alignments.
+> >
+> > This score was among the first to be used for looking up matching spectra in spectral libraries and, to this day, remains one of the most popular scoring methods for both library matching and molecular networking workflows.
+> > <p>&nbsp;</p>
```

Comment:
this should not be necessary

Reply:
Yeah, I don't know how it got there, thanks for noticing. Fixed in [fcd889f](https://github.com/galaxyproject/training-material/pull/3949/commits/fcd889fc380f44e62dc43e45bcd5e3150870903e).